doc/nrf/libraries/modem/lte_lc.rst

```diff
@@ -20,6 +20,9 @@ Configuration
 
 To enable the library, set the Kconfig option :kconfig:option:`CONFIG_LTE_LINK_CONTROL` to ``y`` in the project configuration file :file:`prj.conf`.
 
+.. note::
+   This will enable the core features of the library related to network connectivity. To enable the other features, there are other Kconfig options that are explained later in this chapter.
```

Comment:
```suggestion
   This enables the core features of the library related to network connectivity.
   ```

Reply:
Is this enough? Any other way to stress that enabling the library enables only a limited featureset?

Reply:
How about like this: By default, the library enables only the core features related to the network connectivity.

Reply:
Better, thanks.